internal-webapps/org.wso2.bfsi.authentication.endpoint/src/main/webapp/oauth2_authz_confirm.jsp

```diff
@@ -0,0 +1,46 @@
+<%--
+ ~ Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ ~
+ ~ WSO2 LLC. licenses this file to you under the Apache License,
+ ~ Version 2.0 (the "License"); you may not use this file except
+ ~ in compliance with the License.
+ ~ You may obtain a copy of the License at
+ ~
+ ~     http://www.apache.org/licenses/LICENSE-2.0
+ ~
+ ~ Unless required by applicable law or agreed to in writing,
+ ~ software distributed under the License is distributed on an
+ ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ ~ KIND, either express or implied. See the License for the
+ ~ specific language governing permissions and limitations
+ ~ under the License.
+ --%>
+
+<%@ page import="net.minidev.json.JSONArray" %>
+<%@ page import="net.minidev.json.JSONObject" %>
+<%@ page import="java.util.Map" %>
+<%@ page import="java.util.HashMap" %>
+<%@ page import="org.owasp.encoder.Encode" %>
+<%@ page import="org.apache.commons.lang.StringUtils" %>
+
+<html>
+<head></head>
+<body>
```

Comment:
Why we don't need to include head, footer files into this?

Reply:
Fixed in https://github.com/wso2-extensions/bfsi-consent-management/pull/8/commits/5cab9c9a955547a16e4ab4f4f06ff55db6fb45c5